packages/web3-providers-http/src/index.ts

```diff
@@ -204,7 +204,7 @@ export default class Web3ProvidersHttp
                 throw Error('No HTTP client initiliazed');
 
             const response =
-                args?.ethVersion === 2
+                args.ethVersion === 2
```

Comment:
In multiple places you have `args?.`, but for these methods, `args` is required, so there's no need for the `?`

Reply:
ah i thought this was checking if ethVersion existed. got it